Chaincase.UI/Shared/MainLayout.razor

```diff
@@ -1,22 +1,29 @@
-﻿@inherits LayoutComponentBase
-
+﻿@inherits LayoutComponentBase 
 @using Chaincase.UI.Services
 @using Chaincase.Common
 @implements IDisposable
 @inject UIStateService StateService
+@inject StackService StackService
 @inject NavigationManager NavigationManager
 @inject ThemeSwitcher ThemeSwitcher
 @inject Global Global
 
-<ion-split-pane when="sm" content-id="main-content">
+<ion-split-pane when="sm" content-id="main-content" disabled="@StackService.AnyStackState">
     <ion-menu content-id="main-content">
         <NavMenu/>
     </ion-menu>
     <ion-page class="ion-page" id="main-content">
         <ion-header>
             <ion-toolbar>
                 <ion-buttons slot="start">
-                    <ion-menu-button></ion-menu-button>
+                    @if (StackService.AnyStackState)
+                    {
+                        <ion-back-button @onclick="() => StackService.InvokeStackState()" @onclick:preventDefault="true" default-href="lol"></ion-back-button>
```

Comment:
alternatively `@onclick="StackService.InvokeStackState"`

Reply:
I'm not sure that would work, since onclick wants to a method with `MouseEventArgs`  as a parameter